src/Interfaces/AbstractZonotope.jl

```diff
@@ -426,7 +426,7 @@ this method; otherwise, redundant vertices may be present.
 function vertices_list(Z::AbstractZonotope{N};
                        apply_convex_hull::Bool=true) where {N<:Real}
     c = center(Z)
-    G = remove_zero_columns(genmat(Z))
+    G = genmat(Z)
```

Comment:
This changes the result for `apply_convex_hull=false` in 2D. What is the motivation for this change? I do not see why 2D should behave differently.

```julia
julia> Z1 = Zonotope([0.0, 0.0], zeros(2, 1));
julia> Z2 = Zonotope([0.0, 0.0], zeros(2, 2));
julia> Z3 = Zonotope([0.0, 0.0], zeros(2, 4));

# this branch
julia> vertices_list(Z1; apply_convex_hull=false)
2-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
 [0.0, 0.0]

julia> vertices_list(Z2; apply_convex_hull=false)
4-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]

julia> vertices_list(Z3; apply_convex_hull=false)
8-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]
 [0.0, 0.0]

# master
julia> vertices_list(Z1; apply_convex_hull=false)
1-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
julia> vertices_list(Z2; apply_convex_hull=false)
1-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
julia> vertices_list(Z3; apply_convex_hull=false)
1-element Array{Array{Float64,1},1}:
 [0.0, 0.0]
```

Reply:
the motivation is for performance reasons, because remove_zero_columns doesn't need to be applied in general; in 2d, the convex hull will reduce such vertices efficiently.